### DISCUSSION TITLE
Mealie LDAP Support

### DISCUSSION THREAD

**Discussion by Alsterdetektive1** [mealie-recipes/mealie#1024] [Q&A]
Hey guys,
i want my Mealie to integrate into LDAP. At installation site i found some hints how to do this, with docker but i am missing a bunch of things, maybe someone of you can help.

1. My LDAP has no anonymous access, how can i give mealie an account to check for the data. Maybe over the "LDAP_BIND_TEMPLATE"-Variable? But where to set the password?
2. I only want, that my "mealie" group has access to Mealie. How to define a user filter, who can login? Is it just the "LDAP_ADMIN_FILTER"?

Thanks in advance.

**Comment by Entheologian** [comment]
Since then, did you get this resolved?

**Comment by michael-genson** [comment]
LDAP docs might help here: https://docs.mealie.io/documentation/getting-started/authentication/ldap/
If you're still having trouble with LDAP, feel free to open an issue
